admin_manual/appliance/firewall.rst

```diff
@@ -4,16 +4,18 @@ Firewall protected environment
 If you are considering setting up the appliance in an environment with a firewall, 
 please create rules that permit access to the following hosts. 
 If your DNS is not working, you can use the IP addresses instead.
-If you are using google as your DNS server(IP=8.8.8.8), you have to permit access to it too.
+If you are using Google as your DNS server (IP=8.8.8.8), you have to permit access to it too.
 
 
-Firewall Rules::
+Firewall Rules:
 
-	software-univention.de
-	owncloud.org
-	owncloud.com
-	docker.software-univention.de
-	176.9.114.147
-	marketplace.owncloud.com
-	5.9.68.237
-  	8.8.8.8
+- 8.8.8.8
+- software-univention.de
+- docker.software-univention.de
+- 176.9.114.147
+- updates.software-univention.de
+- 176.9.114.147
```

Comment:
176.9.114.147 is double present, see line 15

Reply:
-.-